### PR TITLE
Feature/upload msixboundle as artifact #AB271

### PR DESCRIPTION
**Pull request by szlatkow**
- Feature - uploading the msixboundle as an artifact.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ jobs:
   build:
     strategy:
       matrix:
-        configuration: [Debug, Release]
+        configuration: [Debug, Release]
```
Comment (aleks-ivanov) [inline .github/workflows/main.yml:9]: remove empty space

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -18,7 +18,14 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 1.11
-
```
Comment (aleks-ivanov) [inline .github/workflows/main.yml:21]: all steps must be indented one tab further in

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -18,7 +18,14 @@ jobs:
...
- 
+
+    - name: Install .NET Core
+      uses: actions/setup-dotnet@v1
+
+    - name: Setup MSBuild
+      id: setup_msbuild
+      uses: microsoft/setup-msbuild@v1
+
     - uses: actions/checkout@v2
```
Comment (aleks-ivanov) [inline .github/workflows/main.yml:29]: checkout action should be as per our conventions

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -95,4 +111,11 @@ jobs:
       shell: powershell
       run: .\.sonar\scanner\dotnet-sonarscanner end /d:sonar.login="${{ secrets.SONAR_TOKEN }}"
 
+    - name: Upload build artifacts
+      continue-on-error: true
```
Comment (aleks-ivanov) [inline .github/workflows/main.yml:115]: remove continue-on-error

File: .github/workflows/main.yml [hunk 5]
```diff
@@ -95,4 +111,11 @@ jobs:
...
+      uses: actions/upload-artifact@v1
+      with:
+       name: MSIX Package
```
Comment (aleks-ivanov) [inline .github/workflows/main.yml:118]: rename artifact zip to something more appropriate

File: .github/workflows/main.yml [hunk 6]
```diff
@@ -6,7 +6,7 @@ jobs:
   build:
     strategy:
       matrix:
-        configuration: [Debug, Release]
+        configuration: [Debug, Release]    
     env:
       Solution_Name: src\Notepads.sln
       Project_Path: src\Notepads\Notepads.csproj
```
Comment (aleks-ivanov) [inline .github/workflows/main.yml:12]: remove Project_Path variable, since it is not used anywhere

Comment (kulov) [reply .github/workflows/main.yml:12]: definitely not

**Review by kulov — CHANGES_REQUESTED** [review]
please address all feedback

**Review by kulov — APPROVED** [review]

**Pull request merged**